lib/screens/committees_screen.dart

```diff
@@ -45,9 +43,8 @@ class _CommitteesScreenState extends State<CommitteesScreen> {
     return CustomScaffold(
       body: ListView(
         children: <Widget>[
-          CustomAppBar(
+          const CustomAppBar(
             title: "Committees & Events",
-            image: Image.asset(ImageAssets.committes),
```

Comment:
Why you removed the image?

Reply:
Its handled in class as I needed custom specs for my image